app/components/timesheet-actions.js

```diff
@@ -1,22 +1,32 @@
 import Component from '@glimmer/component';
-import { task } from 'ember-concurrency';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import constants from 'frontend-timekeeper/constants';
+const { TIMESHEET_STATUSES } = constants;
 
 export default class TimesheetActionsComponent extends Component {
-  get canSubmitHolidays() {
-    return !this.args.timesheet
-      || this.args.timesheet.isDraft;
-  }
+  @tracked absenceComplete = false;
+  @tracked timesheetComplete = false;
 
-  get canSubmitTimesheet() {
-    return !this.canSubmitHolidays
-      && this.args.timesheet.isAbsenceSubmitted;
+  @action
+  onAbsenceClick(checked) {
+    if (!checked) {
+      this.timesheetComplete = false;
+      this.args.onStatusChanged(TIMESHEET_STATUSES.DRAFT);
+    } else {
+      this.args.onStatusChanged(TIMESHEET_STATUSES.ABSENCE_SUBMITTED);
+    }
+    this.absenceComplete = checked;
   }
 
-  submitHolidays = task(async () => {
-    await this.args.onSubmitHolidays();
-  });
-
-  submitTimesheet = task(async () => {
-    await this.args.onSubmitTimesheet();
-  });
+  @action
+  onTimesheetClick(checked) {
```

Comment:
Action handlers are typically named with an active verb like `clickTimesheet`.

Reply:
fixed in [38b50ea](https://github.com/redpencilio/frontend-timekeeper/pull/4/commits/38b50eaaf61b2195024c3afdce974fef5d636e19)